### ISSUE TITLE
Error: image is undefined

### ISSUE THREAD

**Issue by Jarry1250** [hatnote/montage#6]
I get this on one image, I think Engine_Vein%2C_Alderley_Edge_Mines_2016_080.jpg

```
Error: image is undefined
getImageName@https://tools.wmflabs.org/montage-beta/dist/bundle.min.js?v=16.10.12:72:25846
anonymous/fn@https://tools.wmflabs.org/montage-beta/dist/bundle.min.js?v=16.10.12 line 13 > Function:2:395
inputsWatchDelegate/<@https://tools.wmflabs.org/montage-beta/dist/bundle.min.js?v=16.10.12:10:863
$RootScopeProvider/this.$get</Scope.prototype.$digest@https://tools.wmflabs.org/montage-beta/dist/bundle.min.js?v=16.10.12:10:16748
$RootScopeProvider/this.$get</Scope.prototype.$apply@https://tools.wmflabs.org/montage-beta/dist/bundle.min.js?v=16.10.12:10:19215
done@https://tools.wmflabs.org/montage-beta/dist/bundle.min.js?v=16.10.12:9:4355
completeRequest@https://tools.wmflabs.org/montage-beta/dist/bundle.min.js?v=16.10.12:9:8221
createHttpBackend/</xhr.onload@https://tools.wmflabs.org/montage-beta/dist/bundle.min.js?v=16.10.12:9:9042
https://tools.wmflabs.org/montage-beta/dist/bundle.min.js?v=16.10.12
Line 9
```

**Comment by mahmoud** [comment]
Thanks for the report. Pawel works on the frontend and will be taking a look shortly. We'll keep you updated!

**Comment by yarl** [comment]
Should be fixed by bd9c3491ad3c837f4e040307845527c8515b985f. Please reopen if persist.
